driver-kafka/pom.xml

```diff
@@ -62,7 +76,17 @@
         </dependency>
 
     </dependencies>
-
+    <repositories>
```

Comment:
It's best to avoid adding vendor-specific repositories to the project where we can. Is io.confluent:kafka-avro-serializer  available in any mainstream repository?

Reply:
I do recognize that we'll need to make exceptions to this where necessary, but the goal is to avoid adding dependencies to the build where possible. As a local dependency in the kafka module, it's not a huge impact.

Reply:
@jshook Redhat maven repository seems to have an old version which is not working for me. Couldn't find it anywhere else.

Reply:
Understood. We'll leave it in here, as this is incremental functionality and it is module and protocol-specific. I don't see a better alternative. We need to add something about avoiding repo dependencies into the dev guide just to clarify the reasoning, though.